tests/NHSD.BuyingCatalogue.SolutionLists.Application.UnitTests/SolutionListTests.cs

```diff
@@ -46,7 +47,7 @@ public void ShouldListSolutionsWithNoCapabilityFilter()
             repositorySolutions.AddRange(GetSolutionWithCapabilities("S2", "Sup1", false, 2, 3));
             _context.MockSolutionListRepository.Setup(r => r.ListAsync(false, It.IsAny<CancellationToken>())).ReturnsAsync(repositorySolutions);
 
-            var solutions = _context.ListSolutionsHandler.Handle(new ListSolutionsQuery(), new CancellationToken());
+            var solutions = _context.ListSolutionsHandler.Handle(new ListSolutionsQuery(new ListSolutionsFilterViewModel()), new CancellationToken());
```

Comment:
This is associated with the reference to the API in the previous comment. Here we could mock the interface passed into the query.

Reply:
Love it, mocks will make it cleaner, good shout!